lib/padding.m

```diff
@@ -0,0 +1,12 @@
+function [T,F] = padding(T,F,lu1,lu2,LU1,LU2)
```

Comment:
i think that this function could be generalised to something like:
```
function [T,F]=PaddingSignals(t1,f1,t2,f2)
   if ( size(t1,2)>size(t1,1) )
      T=t1'; % transpose array/matrix
   else
      T=t1;
   end
   if ( size(f1,2)>size(f1,1) )
      F=f1'; % transpose array/matrix
   else
      F=f1;
   end
   if ( size(t2,2)>size(t2,1) )
      T(1:length(t2),size(T,2)+1)=t2'; % transpose array/matrix
   else
      T(1:length(t2),size(T,2)+1)=t2;
   end
   if ( size(f2,2)>size(f2,1) )
      F(1:length(f2),size(F,2)+1)=f2'; % transpose array/matrix
   else
      F(1:length(f2),size(F,2)+1)=f2;
   end
end
```

This function could then be used as:
```
[t,f]=PaddingSignals(t1,f1,t2,f2);
[t,f]=PaddingSignals(t,f,t3,f3);
[T,F]=PaddingSignals(lu,LU,lu1,LU1);
[T,F]=PaddingSignals(T,F,lu2,LU2);
```